network/hybridNetwork.go

```diff
@@ -48,7 +47,14 @@ func NewHybridP2PNetwork(log logging.Logger, cfg config.Local, datadir string, p
 	p2pcfg.NetAddress = cfg.P2PHybridNetAddress
 	p2pcfg.IncomingConnectionsLimit = cfg.P2PHybridIncomingConnectionsLimit
 	identityTracker := NewIdentityTracker()
-	p2pnet, err := NewP2PNetwork(log, p2pcfg, datadir, phonebookAddresses, genesisID, networkID, nodeInfo, &identityOpts{tracker: identityTracker})
+
+	var p2pDummyMeshCreator *dummyMeshCreator
+	p2pMeshCreator := meshCreator
+	if p2pMeshCreator == nil && cfg.NetAddress != "" {
```

Comment:
I never know what's going on with all the checks to see if certain addresses are blank or not.  There's some convention that certain addresses turn on different networks.  It only appears once here, but I recall being confused by this in previous PRs as well.  Can we can make some accessors like `cfg.WSConfigured()`, `cfg.P2PConfigured()` or similar?